package.json

```diff
@@ -11,31 +11,30 @@
     "vow-fs": "^0.3.4"
   },
   "devDependencies": {
-    "bem-components": "bem/bem-components#v2.1.0",
```

Comment:
Do you really want to keep `bower` in `islands` library? I heard that you've thought about taking it away. Am I wrong?